src/js/remoteServices.js

```diff
@@ -88,21 +82,15 @@ export default {
     description: (document, service, { title }) =>
       document.querySelector(".js-title-helper")?.textContent?.trim() || title,
     projectId: document =>
-      document
-        .querySelector(".js-title-helper")
-        ?.textContent?.trim()
-        ?.match(projectRegex)?.[1],
+      projectIdentifierBySelector(".js-title-helper")(document) ||
```

Comment:
Ok I was confused because imho it does not make any sense to put the Project-Identifier in every card's title. But maybe there are use cases for a lot of smaller projects...